### PR TITLE
Popup Window

### PR DESCRIPTION
**Pull request by audukehinde**
# In this project I implemented the popup window and it has the following features:
##  I also generated my project section with Javascript

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,68 +1,174 @@
-const hamburger = document.querySelector('.hamburger');
-const navMenu = document.querySelector('.nav-menu');
-const close = document.querySelector('.close');
-const open = document.querySelector('.fa-bars');
-const navLink = document.querySelectorAll('.nav-link');
-// const btnProject = document.querySelector('.btnsee');
-// Initializing our body
-// const bodyElement = document.body;
+document.addEventListener('DOMContentLoaded', () => {
```
Comment (carlos-ssh) [inline main.js:1]: - Kindly enable JS linters(ESLint) in your pull request.

Comment (carlos-ssh) [reply main.js:1]: - Assign a License to your project.

File: main.js [hunk 2]
```diff
@@ -1,68 +1,174 @@
...
+    localStorage.setItem('myFormData', JSON.stringify(myFormData));
+  };
+  form.addEventListener('change', addDataToLocalStorage);
 
-function toggleDisplay(element, val) {
-  element.style.display = val;
-}
+  let getLocalStorageData = localStorage.getItem('myFormData');
+  getLocalStorageData = JSON.parse(getLocalStorageData);
+  document.getElementById('full-name').value = getLocalStorageData.fullName;
```
Comment (carlos-ssh) [inline main.js]: Kindly fix this issue related to this line:
Screen Shot 2021-10-12 at 15 11 25

[Live version](https://raw.githack.com/audukehinde/portfolio/Dom-manipulation/index.html)

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hello Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
changesrequested



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 2 above)
Comment (audukehinde) [reply main.js]: Hello Carlos. Thanks for your kind review. I'm not getting this particular error at my end. How do I fix it? Thanks

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Kenny Audu, 
 ### Thank you for working on the changes suggested previously. :ok_hand:

## To Improve 🙏
### But I am unable to see your project's Pop-up section. In fact, no project is showing. Kindly double-check everything is working fine before asking for a review.

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by dasileker — APPROVED** [review]
### **Hi @audukehinde,** 

### **Congratulations! 🎉 Project Approved 💯✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 


Approved
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**